### PR TITLE
Fix: Fixed NullReferenceException with SetSelectedPathOrNavigate

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
https://appcenter.ms/orgs/FilesApp/apps/Files/crashes/errors/103315946u/overview

**Validation**
How did you test these changes?
- [ ] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Click settings button ...

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs [hunk 1]
```diff
@@ -406,15 +406,15 @@ public void SetSelectedPathOrNavigate(PathNavigationEventArgs e)
 					}
 				}
 			}
-			if (PathNormalization.NormalizePath(ParentShellPageInstance.FilesystemViewModel.WorkingDirectory) !=
-				PathNormalization.NormalizePath(e.ItemPath))
-			{
+
+			if (ParentShellPageInstance is null)
+				return;
```
Comment (yaira2) [inline src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs:411]: Should we also call `DismissOtherBlades`?

Comment (hishitetsu) [reply src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs:411]: I don't think so.

**Review by hishitetsu — APPROVED** [review]
LGTM

**Comment by yaira2** [comment]
This seems to still be an issue https://appcenter.ms/orgs/FilesApp/apps/Files/crashes/errors/2907191234u/overview

File: src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs [hunk 2]
```diff
@@ -406,15 +406,15 @@ public void SetSelectedPathOrNavigate(PathNavigationEventArgs e)
...
+
+			if (NormalizePath(ParentShellPageInstance.FilesystemViewModel.WorkingDirectory) !=
```
Comment (hishitetsu) [inline src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs:413]:
```suggestion
			if (NormalizePath(ParentShellPageInstance.FilesystemViewModel?.WorkingDirectory) !=
```

Comment (yaira2) [reply src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs:413]: Do you want to open a PR for this?

Comment (hishitetsu) [reply src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs:413]: I'm out of home right now so I can't open a PR right away. If you can, I will leave it to you.

Comment (yaira2) [reply src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs:413]: 👍

**Pull request merged**